R/showcase.R

```diff
@@ -105,12 +105,11 @@ navTabsDropdown <- function(files) {
 tabContentHelper <- function(files, path, language) {
   lapply(files, function(file) {
     with(tags,
-      div(class=paste("tab-pane",
+      div(class=paste0("tab-pane",
+                      # TODO: what if the app filename is something else?
                       if (tolower(file) %in% c("app.r", "server.r")) " active"
-                      else "",
-                      sep=""),
-          id=paste(gsub(".", "_", file, fixed=TRUE),
-                   "_code", sep=""),
+                      else ""),
```

Comment:
Nit: I found this logic a little hard to follow. It may be idiomatic and I haven't seen it yet, but I would have found an `ifelse` more readable (or just pulling it out into a separate variable elsewhere)

Reply:
I tend to avoid `ifelse()` for coercion reasons. That being said, I don't like the nested `if` `else` logic here, but the rest of `showcase.R` is littered with that pattern so I feel like that could be for another PR

Reply:
Fair point! Let's leave it as-is, then.